ionic_demo/plugins/ft_sensor_determinism/README.md

```diff
@@ -0,0 +1,75 @@
+# Force-torque sensor determinism
+
+The Force-torque wrench data can now be read directly from the ECM in
+addition to subscribing to a gz-transport topic.
+This example plugin can be attached to a force-torque sensor to compare the
+determinism of data access via these two methods by printing a console error
+message whenever old data is accessed.
+
+## Build
+
+From this folder, do the following to build the plugin:
+
+~~~
+mkdir build
+cd build
+cmake ..
+make
+~~~
+
+This will generate the `FTSensorDeterminism` library under `build`.
+
+## Run
+
+A demo world is adapted from the
+[mimic\_fast\_slow\_pendulums\_world.sdf](https://github.com/gazebosim/gz-sim/blob/gz-sim9/examples/worlds/mimic_fast_slow_pendulums_world.sdf)
+example world added to demonstrate mimic constraints in Gazebo Harmonic.
+A force-torque sensor with an instance of the `FTSensorDeterminism` plugin is
+added to each joint in the world.
+
+Before starting Gazebo, we must make sure it can find the plugin by doing:
+
+~~~
+cd ionic_demo/plugins/ft_sensor_determinism
+export GZ_SIM_SYSTEM_PLUGIN_PATH=`pwd`/build
+~~~
+
+Then run the demo world:
+
+    gz sim -v 3 priority_printer_plugin.sdf -r
```

Comment:
should the world sdf name be `ft_sensor_determinism.sdf`?

Reply:
thanks, fixed in https://github.com/gazebosim/ionic_demo/pull/10/commits/d205d4f33806e66595a49ed95b484d6f3ecdbb42